src/seedu/addressbook/parser/Parser.java

```diff
@@ -227,7 +227,7 @@ private int parseArgsAsDisplayedIndex(String args) throws ParseException, Number
      * @param args full command args string
      * @return the prepared command
      */
-    private Command prepareFind(String args) {
+    private static Command prepareFind(String args) {
```

Comment:
Don't forget to change the code where this method is used. It should be changed to `Parser.parseCommand(...)` in `Main` class and `ParserTest` class.